### DISCUSSION TITLE
My ideas

### DISCUSSION THREAD

**Discussion by Gorilli09** [Wohlstand/TheXTech#630] [Ideas]
# More control options
## In-Game
* Camera Left
* Camera Right
## Menu
* Up
* Down
* Left
* Right
* Confirm
* Back
* UI Action 1
* UI Action 2
* Previous Tab
* Next Tab
# Video settings
* 16:9 and 16:10 Widescreen
* Resolution settings
# Power-Ups
* Frog Suit
* Superball
* Cape Feather
* P-Balloon
* Boomerang Flower
# Other
* New character: Toadette
* Choosable playstyles (SMBX, SMB1, SMB2, SMB3, SMW)
* Camera doesn't move during level path revelation
* Game pauses during power-up transformation and player death
* Custom character slots
* Timer
* Player voice

**Comment by LoveBodhi** [comment]
Also SMB Wonder, which use LUA to handle.

> 16:9 and 16:10 Widescreen

A part of MuitiRes system

Anyways, some of the part, such as playstyle, should be a part of "Asset Pack" topic...

> SMGW

What's SMGW?
